fastlane/Fastfile

```diff
@@ -543,58 +595,56 @@ platform :ios do
-    upload_symbols_to_crashlytics(
-      app_id: "1:921154746561:ios:216bd10ccc947659027c40",
-      dsym_path: ENV[KEY_DSYM_PATH],
-      gsp_path: "./ios/GoogleService-Info.plist",
-      binary_path: "./ios/Pods/FirebaseCrashlytics/upload-symbols"
-    )
   end
 
-  desc "Upload HybridApp to TestFlight"
-    lane :upload_testflight_hybrid do
-      upload_to_testflight(
-        app_identifier: "com.expensify.expensifylite",
-        api_key_path: "./ios-fastlane-json-key.json",
-        distribute_external: true,
-        notify_external_testers: true,
-        reject_build_waiting_for_review: true,
-        changelog: "Thank you for beta testing New Expensify, this version includes bug fixes and improvements.",
-        groups: ["Applause", "Beta Testers", "Expensify Employees"],
-        demo_account_required: true,
-        beta_app_review_info: {
-          contact_email: ENV["APPLE_CONTACT_EMAIL"],
-          contact_first_name: "Andrew",
-          contact_last_name: "Gable",
-          contact_phone: ENV["APPLE_CONTACT_PHONE"],
-          demo_account_name: ENV["APPLE_DEMO_EMAIL"],
-          demo_account_password: ENV["APPLE_DEMO_PASSWORD"],
-          notes: "1. In the Expensify app, enter the email 'appletest.expensify@proton.me'. This will trigger a sign-in link to be sent to 'appletest.expensify@proton.me'
-                  2. Navigate to https://account.proton.me/login, log into Proton Mail using 'appletest.expensify@proton.me' as email and the password associated with 'appletest.expensify@proton.me', provided above
-                  3. Once logged into Proton Mail, navigate to your inbox and locate the email triggered in step 1. The email subject should be 'Your magic sign-in link for Expensify'
-                  4. Open the email and copy the 6-digit sign-in code provided within
-                  5. Return to the Expensify app and enter the copied 6-digit code in the designated login field"
-        }
-      )
+ desc "Upload DSYMs to Firebase for HybridApp"
+ lane :upload_dsyms do
+   puts "dsym path: #{ENV[KEY_DSYM_PATH]}"
+   upload_symbols_to_crashlytics(
+     app_id: "1:921154746561:ios:216bd10ccc947659027c40",
+     dsym_path: ENV[KEY_DSYM_PATH],
+     gsp_path: "./ios/GoogleService-Info.plist",
+     binary_path: "./ios/Pods/FirebaseCrashlytics/upload-symbols"
+   )
+ end
 
-    puts "dsym path: #{ENV[KEY_DSYM_PATH]}"
-    upload_symbols_to_crashlytics(
-      app_id: "1:1008697809946:ios:3ffad71f664f2886",
-      dsym_path: ENV[KEY_DSYM_PATH],
-      gsp_path: "./ios/GoogleService-Info.plist",
-      # Assuming we are running this from the react-native submodule directory for HybridApp
-      binary_path: "./Mobile-Expensify/iOS/Pods/FirebaseCrashlytics/upload-symbols"
+  desc "Upload HybridApp to TestFlight"
+  lane :upload_testflight_hybrid do
+    upload_to_testflight(
+      app_identifier: ENV["APPLE_HYBRID_APP_ID"],
+      api_key_path: "./ios-fastlane-json-key.json",
+      distribute_external: true,
```

Comment:
Could we set this to false for CP to prod to prevent TF betas from interrupting QA?

Reply:
Yes, we can set this to false for CP to prod and it won't send to groups.

Reply:
nice! We'll have to keep that in mind